### ISSUE TITLE
rustc completion spews about -Z

### ISSUE THREAD

**Issue by codl** [fish-shell/fish-shell#4888]
the first time that an argument to `rustc` is typed, ```error: the option `Z` is only accepted on the nightly compiler``` is dumped to the terminal. other `rustc` commands in the same session don't have that problem

```
$ fish --version
fish, version 2.7.1
$ rustc --version
rustc 1.25.0
```

[asciinema recording](https://asciinema.org/a/6TIWSYNOKailqiZkUjgqCslbD)

**Comment by faho** [comment]
Already fixed by e1c90cac8872f68867e175ec4b4f5379dbfa27b1, which will be part of the next release.

Sorry for the inconvenience!

**Comment by codl** [comment]
oh sweet, thank you!
